docker_images/span_marker/requirements.txt

```diff
@@ -1,4 +1,4 @@
 starlette==0.27.0
 api-inference-community==0.0.25
 huggingface_hub==0.11.0
-span_marker>=1.0.0
\ No newline at end of file
+span_marker>=1.4.0
```

Comment:
You will need to also update the `huggingface_hub` version above, otherwise it won't build (as per the GA errors)

Reply:
I see! I've incremented both the `huggingface_hub` and `api-inference-community` versions.